store/rediss/token.go

```diff
@@ -19,10 +20,29 @@ func NewTokenStore(db *redis.Client) store.TokenStore {
 	}
 }
 
+// Idea so far
+/*
+
+Creating refresh token jti when creating the access token
+So the next time user ask for refresh token, that refresh token jti will be its id
+
+Than store the access token in redis as "access_token:access uid" is the key
+Then store the refresh token in redis as "refresh_token:access uid" is the key
+
+That way, i can remove the other session without knowing the refresh token jti
+
+The loophole is when user has not request for refresh token, the access token is already have its refresh token jti
+This issue can be solved by creating another function to check if the access token has been updated or not
```

Comment:
oh I see, so this is why you need to check for `isUpdated`. That check is rather implicit, and thus error prone. Can we instead check if `refresh_token:access` key actually exists in redis?

Reply:
Yes i was thinking to check it in redis instead of db as well